site/docs/multi-engine-support.md

```diff
@@ -93,7 +93,7 @@ Users should continuously upgrade their Flink version to stay up-to-date.
 | 1.17    | Deprecated      | 1.3.0                   | 1.6.0                  | [iceberg-flink-runtime-1.17](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-flink-runtime-1.17/{{ icebergVersion }}/iceberg-flink-runtime-1.17-{{ icebergVersion }}.jar) |
 | 1.18    | Maintained      | 1.5.0                   | {{ icebergVersion }}   | [iceberg-flink-runtime-1.18](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-flink-runtime-1.18/{{ icebergVersion }}/iceberg-flink-runtime-1.18-{{ icebergVersion }}.jar) |
 | 1.19    | Maintained      | 1.6.0                   | {{ icebergVersion }}   | [iceberg-flink-runtime-1.19](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-flink-runtime-1.19/{{ icebergVersion }}/iceberg-flink-runtime-1.19-{{ icebergVersion }}.jar) |
-| 1.20    | Maintained      | 1.7.0                   | {{ icebergVersion }}   | [iceberg-flink-runtime-1.20](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-flink-runtime-1.20/{{ icebergVersion }}/iceberg-flink-runtime-1.20-{{ icebergVersion }}.jar) |
```

Comment:
`icebergVersion` currently points to `1.6.1` where there's no Flink 1.12 support.

Reply:
@manuzhang: do we release the docs and the code independently? What is the process of updating the docs for the next release then?

Reply:
this doc is under site, which will be deployed immediately after commit